e2e-network/test-03-private-data.sh

```diff
@@ -30,7 +30,7 @@ waitForContainer() {
 }
 
 waitForChaincode() {
-  sh "$TEST_TMP/../wait-for-chaincode.sh" "$1" "$2" "$3" "$4" "$5"
+  sh "$TEST_TMP/../wait-for-chaincode-v1.sh" "$1" "$2" "$3" "$4" "$5"
```

Comment:
`wait-for-chaincode-v1` is added for HLF 2.x cases ? If so we can name it better. I had to do some revere engineering to figure it out.
Maybe `wait-for-chaincode-hlf14` would be better?